newui/docs/_docs/1_4_powered_by.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Talks & Powered By"
+keywords: hudi, talks, presentation
+permalink: /docs/powered_by.html
+---
+
+## Adoption
+
+### Uber
+
+Apache Hudi was originally developed at [Uber](https://uber.com), to achieve [low latency database ingestion, with high efficiency](http://www.slideshare.net/vinothchandar/hadoop-strata-talk-uber-your-hadoop-has-arrived/32).
+It has been in production since Aug 2016, powering the massive [100PB data lake](https://eng.uber.com/uber-big-data-platform/), including highly business critical tables like core trips,riders,partners. It also 
+powers several incremental Hive ETL pipelines and being currently integrated into Uber's data dispersal system.
+
+### Amazon Web Services
+Amazon Web Services is the World's leading cloud services provider. Apache Hudi is [pre-installed](https://aws.amazon.com/emr/features/hudi/) with the AWS Elastic Map Reduce 
+offering, providing means for AWS users to perform record-level updates/deletes and manage storage efficiently.
+
+### EMIS Health
+
+[EMIS Health][https://www.emishealth.com/] is the largest provider of Primary Care IT software in the UK with datasets including more than 500Bn healthcare records. HUDI is used to manage their analytics dataset in production and keeping them up-to-date with their upstream source. Presto is being used to query the data written in HUDI format.
```

Comment:
`[EMIS Health][https://www.emishealth.com/]` should be `[EMIS Health](https://www.emishealth.com/)`? cc @lamber-ken

Reply:
👍 done.